release/ray_release/command_runner/job_runner.py

```diff
@@ -56,11 +57,7 @@ def prepare_local_env(self, ray_wheels_url: Optional[str] = None):
 
     def _copy_script_to_working_dir(self, script_name):
         script = os.path.join(os.path.dirname(__file__), f"_{script_name}")
-        if os.path.exists(script_name):
-            os.unlink(script_name)
-        if os.path.islink(script):
-            script = os.readlink(script)
-        os.link(script, script_name)
+        shutil.copy(script, script_name)
```

Comment:
feels that the "script" that is being copied should be bazel runfiles now.

Reply:
yeah i can do that in a following up PR